grabbit/src/main/groovy/com/twcable/grabbit/server/batch/steps/jcrnodes/JcrNodesReader.groovy

```diff
@@ -33,19 +32,39 @@ import javax.jcr.Node as JcrNode
 @SuppressWarnings("GrMethodMayBeStatic")
 class JcrNodesReader implements ItemReader<JcrNode> {
 
+    private String excludePaths
+    private String[] excludePathList
+    private static final String[] EMPTY_ARRAY = new String[0];
+
+    public void setExcludePaths(String excludePaths) {
+        this.excludePaths = excludePaths
```

Comment:
This is similar to Workflow Configs I pass them from the xml , i.e., 

<bean id="serverJcrNodesReader" class="com.twcable.grabbit.server.batch.steps.jcrnodes.JcrNodesReader" scope="step">
    	<property name="excludePaths" value="#{jobParameters['excludePaths']}"/>
    </bean>

So that is the reason I have the setter